### DISCUSSION TITLE
Is the Macaque Audio Classification project closed?

### DISCUSSION THREAD

**Discussion by philipkd** [earthspecies/library#7] [General] [answered]
I noticed that the [error rate on the leaderboard](https://github.com/earthspecies/library/tree/master/macaques) is now 0.002745. Does that mean the project is effectively finished?

**Comment by radekosmulski** [comment, marked as answer]
Not necessarily. The idea behind the collaboration was to show / test various ways of working with the data. If you have a technique that had not been included yet and would like to make a contribution this would be greatly appreciated! Should someone be working on a similar problem they can come here and will have a collection of methods they can leverage. This particular dataset proved to be not too much of a challenge for modern ML techniques hence the error rate.
